loom/common/platform/platformHttpOSX.mm

```diff
@@ -107,13 +107,13 @@ -(void)connectionDidFinishLoading:(NSURLConnection *)connection
     if(cacheToFile != NULL)
         [receivedData writeToFile:[NSString stringWithUTF8String:cacheToFile] atomically:YES];
 
-    NSString *response = nil;
-    response = [[NSString alloc] initWithData:receivedData encoding:NSUTF8StringEncoding];
-
+    utByteArray* bytes = new utByteArray();
+    bytes->attach(receivedData.mutableBytes, receivedData.length);
```

Comment:
Does it need to get copied instead here or is just reusing the receivedData ok? It seems to work fine as is, but maybe it just has a low repro rate.